iocore/eventsystem/I_IOBuffer.h

```diff
@@ -58,6 +58,7 @@ class VIO;
 inkcoreapi extern int64_t max_iobuffer_size;
 extern int64_t default_small_iobuffer_size;
 extern int64_t default_large_iobuffer_size; // matched to size of OS buffers
+extern int iobuffer_advice;
```

Comment:
@jpeach I'd like to land this as is and write a JIRA to come back to the global variable once we've had time to evaluate this.  Are you okay with this?